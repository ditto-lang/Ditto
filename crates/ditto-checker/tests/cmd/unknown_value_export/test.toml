bin.name = "ditto-checker-testbin-check-module"
args = ["unknown_value_export.ditto"]
fs.sandbox = true
status = "failed"
