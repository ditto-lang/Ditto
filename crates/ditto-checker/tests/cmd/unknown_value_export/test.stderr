
  × unknown value export
   ╭─[unknown_value_export.ditto:1:1]
 1 │ module Test exports (wut);
   ·                      ─┬─
   ·                       ╰── this value isn't defined?
   ╰────

